core/src/main/java/org/apache/iceberg/ManifestFilterManager.java

```diff
@@ -71,6 +72,7 @@ public String partition() {
   private final PartitionSet deleteFilePartitions;
   private final PartitionSet dropPartitions;
   private final CharSequenceSet deletePaths = CharSequenceSet.empty();
+  private final FilesToDeleteHolder filesToDelete = new FilesToDeleteHolder();
```

Comment:
I'd consider calling it `Set<F> deleteFiles` to follow the existing naming convention: 

```
deleteFilePartitions -> a set of partitions of all files marked for deletion
deletePaths -> a set of paths to remove
deleteExpression -> an expression
```

Reply:
as discussed on Slack last week I renamed this class to `PendingDeleteFile` which implements `DeleteFile` (I just didn't want to make this PR too large).

Reply:
makes sense, updated